src/Application/Kernel.php

```diff
@@ -49,9 +53,9 @@ private function createContainer(): Container
             ->config($this->appConfig)
             ->singleton(self::class, fn () => $this)
             ->singleton(Container::class, fn () => $container)
-            ->addInitializer(new RequestInitializer())
-            ->addInitializer(new RouteBindingInitializer())
-            ->addInitializer(new PDOInitializer());
+            ->addInitializer(RequestInitializer::class)
+            ->addInitializer(RouteBindingInitializer::class)
+            ->addInitializer(PDOInitializer::class);
```

Comment:
Somewhat unrelated, but is there a reason we don't move these to a bootstrapper?

Reply:
https://github.com/tempestphp/tempest-framework/issues/189